client/client.go

```diff
@@ -43,6 +43,13 @@ func Dial(rawurl string) (Client, error) {
 	}, nil
 }
 
+func NewClient(rc *rpc.Client) Client {
```

Comment:
what's the difference/purpose of having both `Client`/`c` ?

Reply:
1. the rpc.client is not exposed (https://github.com/ethereum/go-ethereum/blob/master/ethclient/ethclient.go#L36).
2. we want reuse the ethclient code.

so we need to handle `Client`/`c`